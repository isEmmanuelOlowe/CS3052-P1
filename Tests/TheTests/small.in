20
8.40485369e-11 3.96464774e-11
4.46583435e-11 3.53336097e-11
8.86428433e-11 3.18692772e-11
5.8409022e-11 1.55828494e-12
3.83715875e-11 1.59368627e-11
5.88589136e-12 6.91004373e-11
1.63545951e-11 8.99854306e-11
5.33064714e-11 1.59071503e-11
5.82699021e-11 6.0414419e-11
3.90478195e-11 2.69971118e-11
7.42377406e-11 2.9340057e-11
7.55380785e-12 2.98525606e-11
8.57377943e-11 4.04982634e-11
6.6283066e-11 9.41968323e-11
2.75508143e-13 8.4647578e-11
5.32596024e-11 4.62379245e-11
2.65612235e-11 7.87876621e-11
3.06785131e-11 9.82752263e-11
6.08715653e-11 6.00855136e-11
8.85895131e-11 2.12438798e-11
