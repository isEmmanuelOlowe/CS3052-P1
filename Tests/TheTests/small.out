1.89185886e-12
